### DISCUSSION TITLE
Samsung S22 - AR is not tracking (WebXR)

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4155] [Q&A]
Not sure if its issue or not so ill post here.

WebXR, Chrome, Samsung S22.

In AR view getting the message "AR is not tracking!"
Tested and working on Iphone XR, Iphone 8, Samsung S9, Samsung Galaxy TabS4.

**Comment by elalish** [comment]
100% repro? AR is generally finicky - you need to have some texture for the camera to find and move the phone side-to-side. Can you post a screen recording?

**Reply by danilokrulj** [reply]
It was tested by a coworker so I cant screen record at the moment, I will try to obtain a screen record from him.
